passkey-types/src/webauthn/attestation.rs

```diff
@@ -603,6 +606,11 @@ pub struct CollectedClientData {
     #[serde(default, serialize_with = "truthiness")]
     pub cross_origin: Option<bool>,
 
+    /// CollectedClientData can be extended by the user of this library, this accounts for
+    /// keys that are unknown to the library, but may be known to the user.
+    #[serde(flatten)]
```

Comment:
Did you end up testing if having two `serde(flatten)` was okay for serialization/deserialization? Might be a good idea to add some unknown_keys to some of the tests to make sure they survive.

Reply:
Ah yes, good point, I wanted to add tests for it but completely forgot! Fixed!